api/components/attributes/mutual-funds/index.yaml

```diff
@@ -500,13 +500,13 @@ select:
             description: Time associated for recurring orders
           schedule:
             _description:
-              required: optional
+              required: mandatory
```

Comment:
for SIP orders, in `on_select`, `item.time` attribute is provided. So, only if `item.time` is provided, then `schedule` and `frequency` is mandatory. If `item.time` is not provided, `schedule` and `frequency` shouldn’t be checked. But currently the atrribute check is happening and the following error is thrown

```
Error validating examples with attributes Error: Key path time.schedule.frequency does not exist in the example object at select
    at handleError (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:357:9)
    at checkKeysExistence (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:387:9)
    at handleIfObjectIsArray (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:404:9)
    at checkKeysExistence (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:395:7)
    at validateObject (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:351:3)
    at validateExamplesAttributes (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:456:23)
    at async getSwaggerYaml (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:323:7)
    ```